backend/hitas/tests/factories/indices.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from factory import fuzzy
+from factory.django import DjangoModelFactory
+from faker import Faker
+
+from hitas.models import (
+    ConstructionPriceIndex,
+    ConstructionPriceIndexPre2005,
+    MarketPriceIndex,
+    MarketPriceIndexPre2005,
+    MaxPriceIndex,
+)
+
+faker = Faker(locale="fi_FI")
+
+
+class AbstractIndexFactory(DjangoModelFactory):
+    class Meta:
+        django_get_or_create = ("value",)
+        abstract = True
+
+    month = fuzzy.FuzzyDate(start_date=datetime.date(1990, 1, 1))
```

Comment:
It could be an issue that the `month` value is not always the first day of a month, since when the `month` is serialised the day is dropped off and duplicate `month` values may be returned from the endpoint 🤔

Reply:
true. will fix

Reply:
fixed